### DISCUSSION TITLE
Upgrade Tools - Less Clicks

### DISCUSSION THREAD

**Discussion by UGG5721** [sunflower-land/sunflower-land#2589] [Ideas]
Tried of clicking so much to chop trees or break rocks? Me too! Let's get an upgrade to our tools! Similar to how Stardew Valley has upgraded tools, we should be able to upgrade the tools either by obtaining a nft that makes our tools stronger or a upgrade to our blacksmith shop.

Tools can be upgraded to bronze followed by gold. Thoughts?

**Comment by NotKeen** [comment]
Love it! I'd definitely pay to have 1 click per tree chopping and 1 click per node metal mining.
